dropwizard-migrations/src/test/java/io/dropwizard/migrations/DbMigrateCommandTest.java

```diff
@@ -61,10 +61,10 @@ void testDryRun() throws Exception {
         final ByteArrayOutputStream baos = new ByteArrayOutputStream();
         migrateCommand.setOutputStream(new PrintStream(baos));
         migrateCommand.run(null, new Namespace(Collections.singletonMap("dry-run", true)), conf);
-        assertThat(baos.toString(UTF_8.name())).startsWith(String.format(
-                "-- *********************************************************************%n" +
-                "-- Update Database Script%n" +
-                "-- *********************************************************************%n"));
+        assertThat(baos.toString(UTF_8.name())).isEqualToNormalizingNewlines(
+                "-- *********************************************************************\n" +
+                "-- Update Database Script\n" +
+                "-- *********************************************************************\n");
```

Comment:
Incorrect replacement (starts with vs. equals).

Reply:
Right you are - fun that the test still passed!